bundles.go

```diff
@@ -54,7 +47,7 @@ func (b Bundles) hasDuplicatedVersions() bool {
 		var seen int
 
 		for _, b2 := range b {
-			if b1.Version == b2.Version && b1.Provider == b2.Provider {
+			if b1.Version == b2.Version && b1.Provider == b2.Provider && b1.Name == b2.Name {
```

Comment:
Change makes sense but I don't know how to test this. The issue is whether it causes problems with the changelogs we display to customers. @tuommaki WDYT?

Reply:
Hmm, I don't think changelogs would affect this problem.

Reply:
This library is used to format the changelogs we show in gsctl and Happa.

e.g.

```
gsctl list releases
gsctl show release 6.2.0
```